src/particleContainer/TraversalTuner.h

```diff
@@ -263,9 +276,9 @@ void TraversalTuner<CellTemplate>::readXML(XMLfileUnits &xmlconfig) {
 }
 
 template<class CellTemplate>
-void TraversalTuner<CellTemplate>::rebuild(std::vector<CellTemplate> &cells, 
-        const std::array<unsigned long, 3> &dims) {
-    
+void TraversalTuner<CellTemplate>::rebuild(std::vector<CellTemplate> &cells,
+        const std::array<unsigned long, 3> &dims, double cellLength[3], double cutoff) {
```

Comment:
why not use std::array?

Reply:
because cellLength is always double[3] and would need to be changed everywhere...